airbyte-integrations/connector-templates/source-configuration-based/setup.py.hbs

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk~=0.1.64",
```

Comment:
is there a way to always get the latest version when running the generator?

Reply:
Do we always want the latest bumps in major versions too? The `~=` should continue to get the latest patch. I can see extending this out to the minor version, but we may still want to be restrictive against getting the latest when theres a major version bump.

Reply:
I think it's fine to freeze the requirements, but i would expect the generator to create a connector using the latest version of the cdk